docs/appcircle-api/faq.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Troubleshooting & FAQ
+description: Troubleshooting & FAQ
+tags: [api, authentication, personal api token, api token, session token, troubleshooting, faq]
+sidebar_position: 5
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Overview
+
+This section is designed to help you quickly find answers to common questions and provide you with a better understanding of Appcircle server API.
+
+## Appcircle Server API FAQ
+
+### How can I check the versions of an app published in the Enterprise App Store and download it?
+
+After you have successfully [created the Personal API Token](/appcircle-api/api-authentication.md#generatingmanaging-the-personal-api-tokens) from the **root organization**, you can check the versions of an application and download them, like in the following example code blocks.
+
+:::caution
+You must create the PAT on the root organization. Requests will fail if you create the PAT from a sub-organization.
```

Comment:
Referring to Personal API Token as PAT may cause confusion with Personal Access Token. Should we change it or keep it for simplicity?

Reply:
Added what we mean by "PAT (Personal API TOKEN)". 
User now can easily understand when we sat PAT in the rest of the documentation, we mean "Personal API Token".